scripts/metrics.py

```diff
@@ -1,124 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Since this was removed, the invoke task will also need to be changed to remove this module. I will do this when I merge.